src/screens/ProfileScreen/index.js

```diff
@@ -73,13 +74,12 @@ import BottomSheetBio from './elements/BottomSheetBio';
 import BottomSheetImage from './elements/BottomSheetImage';
 import BottomSheetRealname from './elements/BottomSheetRealname';
 import FollowInfoRow from './elements/FollowInfoRow';
+import {KarmaLock} from './elements/KarmaLock';
 import {KarmaScore} from './elements/KarmaScore';
 import LinkAndSocialMedia from './elements/LinkAndSocialMedia';
 import ProfileHeader from './elements/ProfileHeader';
 import ProfilePicture from './elements/ProfilePicture';
 import ProfileTiktokScroll from './elements/ProfileTiktokScroll';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statement for `KarmaLock` from `./elements/KarmaLock` has been moved. Make sure that this component is not used before its import.
```diff
- import {KarmaLock} from './elements/KarmaLock';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->